File: less/mario.c
#include <cs50.h>
#include <stdio.h>
int main(void)

{
    //type a integer height between 1 and 23
    int i;
    do
    {
      i = get_int("height: ");
    }
    while ((i <= 0) || (i > 23));

    //print symbol from top row to bottom
    for (int j = 0; j < i; j++)
    {
        //print space in row
        for (int k = 0; k < i - 1 - j; k++)
        {
            printf(" ");
        }
        //print # in row
        for (int h = 0; h < j + 2; h++)
        {
            printf("#");
        }
        //print next line
        printf("\n");
    }
}

File: less/mario.py
# import get_int function from cs50
from cs50 import get_int

# prompt the user to input the height [0,23]
while True:
    height = get_int("height: ")
    if (height >= 0) and (height < 24):
        break
# print the spaces and #s
for i in range(height):
    print(" " * (height - 1 - i), end="")
    print("#" * (i + 2))